src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -56,269 +57,21 @@ static const uint32_t FDT_END = 0x9;
 
 // ------ L O C A L  F U N C T I O N S  A N D  S T R U C T S ------------------
 struct fdt_header {
-  uint32_t magic;
-  uint32_t totalsize;
-  uint32_t off_dt_struct;
-  uint32_t off_dt_strings;
-  uint32_t off_mem_rsvmap;
-  uint32_t version;
-  uint32_t last_comp_version;
-  uint32_t boot_cpuid_phys;
-  uint32_t size_dt_strings;
-  uint32_t size_dt_struct;
+    uint32_t magic;
```

Comment:
Why the extra spaces.  Currently, the code here is using 2 spaces for idention.

Reply:
Removed extra spaces.